lib/Db/ChattyLLM/Message.php

```diff
@@ -78,6 +85,7 @@ public function jsonSerialize() {
 			'timestamp' => $this->timestamp,
 			'ocp_task_id' => $this->ocpTaskId,
 			'sources' => $this->sources,
+			'attachments' => json_decode($this->attachments, true) ?: [],
```

Comment:
This can throw, no? Would be good to add a docblock tag for that and catch it in downstream callers